File: 参考/comp9041-Q/cs9041/lab07/total_words.py
#!/usr/bin/python3
# The function re.split or the function re.findall could be used to separate words. 
import re,sys
sum = 0
for line in sys.stdin:
    l = re.split(r'[^a-zA-Z]',line)
    new_l = []
    for x in l:
    	if x:
    		new_l.append(x)
    sum += len(new_l)
print(sum,end='')
print(" words")

File: 参考/comp9041-Q/cs9041/lab07/total_words.pl
#!/usr/bin/perl -w
while($line = <>){
	@arr = $line =~ /[a-zA-Z]+/g;
	$tmp = 0;
	foreach $x (@arr){
		if ($x){
			$tmp++;
		}
	}
	$count += $tmp;
	# print $tmp," $count";
	# print join":",@arr,"\n";
}
print $count," words\n";